mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -907,6 +905,11 @@ public void CancelErrorRead ()
 				}
 			}
 		}
+
+		void SetProcessHanle (SafeProcessHandle handle) {
```

Comment:
Why don't we use the existing https://github.com/mono/referencesource/blob/mono/System/services/monitoring/system/diagnosticts/Process.cs#L1836 ?